ts/features/walletV3/payment/screens/WalletPaymentConfirmScreen.tsx

```diff
@@ -69,45 +59,47 @@ const WalletPaymentConfirmScreen = () => {
     }, [dispatch])
   );
 
-  React.useEffect(() => {
-    if (pot.isSome(authorizationUrlPot)) {
-      navigation.navigate(WalletPaymentRoutes.WALLET_PAYMENT_MAIN, {
-        screen: WalletPaymentRoutes.WALLET_PAYMENT_OUTCOME
-      });
-    }
-  }, [authorizationUrlPot, navigation]);
-
-  const taxFee = pipe(
-    selectedPspOption,
-    O.chain(psp => O.fromNullable(psp.taxPayerFee)),
-    O.getOrElse(() => 0)
-  );
-
-  const paymentAmount = pipe(
-    pot.toUndefined(paymentDetailsPot),
-    O.fromNullable,
-    O.chain(paymentDetails => O.fromNullable(paymentDetails.amount)),
-    O.getOrElse(() => 0)
-  );
-
-  const selectedMethod = O.toUndefined(selectedMethodOption);
-
-  const selectedPsp = O.toUndefined(selectedPspOption);
-
-  const handleStartPaymentAuthorization = () => {
-    if (selectedMethod && selectedPsp && pot.isSome(transactionPot)) {
-      dispatch(
-        walletPaymentAuthorization.request({
-          paymentAmount: paymentAmount as AmountEuroCents,
-          paymentFees: taxFee as AmountEuroCents,
+  const handleStartPaymentAuthorization = () =>
+    pipe(
+      sequenceS(O.Monad)({
+        paumentDetail: pot.toOption(paymentDetailsPot),
+        transaction: pot.toOption(transactionPot),
+        selectedPsp: selectedPspOption,
+        selectedMethod: selectedMethodOption
+      }),
+      O.map(({ paumentDetail, selectedMethod, selectedPsp, transaction }) =>
```

Comment:
Typo on `paumentDetail`

Reply:
Ops! Fixed in https://github.com/pagopa/io-app/pull/5276/commits/4eed001f22013531ba9735a63fda2c2a6ef968ee